### DISCUSSION TITLE
Initial commits

### DISCUSSION THREAD

**Discussion by Krukrukruzhka** [calculation-methods/M8O-303B-21#1] [Q&A]
Доброго времени суток, Дмитрий Евгеньевич. Мы можем сделать тестовый pull request, чтобы посмотреть, как он работает, а также, чтобы вы проверили оформление README, директории stud, её поддиректорий и репозитория в целом?

**Comment by pivovarov-mai** [comment]
Да. Так и надо. Если есть группа, то я готов присоединить ее к репозиторию.

**Reply by Krukrukruzhka** [reply]
Мы организацию создали, в неë сделали форк от текущего проекта, дополнили папкой stud, отредактировали README. Всех студентов группы в организацию добавили. 
Если это вставать, то завтра после пары можем задержаться и сделать  первый pull request?

**Reply by pivovarov-mai** [reply]
А без создания организации это нельзя было сделать?

**Reply by Krukrukruzhka** [reply]
Все ссылки по запросам о формировании группы/команды на гитхабе, выводили на создание организации, поэтому так и сделали 
Пока это выглядит так https://github.com/M8O-303B-21/M8O-303B-21
